File: ios/Sources/SIT/Views/Shared/Warm/WarmEyebrow.swift
import SwiftUI

/// Small uppercase label used above sections and inside cards.
/// Mirrors `Eyebrow` in system.jsx (lines 406-415).
struct WarmEyebrow: View {
    let text: String
    var warmth: Warmth = .subtle

    var body: some View {
        Text(text.uppercased())
            .font(.system(size: 11, weight: .semibold))
            .tracking(11 * 0.16)
            .foregroundStyle(WarmTheme.palette(for: warmth).ink3)
    }
}

#Preview {
    VStack(alignment: .leading, spacing: 16) {
        WarmEyebrow(text: "Today")
        WarmEyebrow(text: "Upcoming")
        WarmEyebrow(text: "Snoozed")
    }
    .padding()
    .background(WarmTheme.subtle.paper)
}

File: android/app/src/main/java/com/xaymaca/sit/ui/warm/WarmEyebrow.kt
package com.xaymaca.sit.ui.warm

import androidx.compose.foundation.layout.Spacer
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.layout.size
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.text.TextStyle
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp
import com.xaymaca.sit.ui.theme.Warmth
import com.xaymaca.sit.ui.theme.WarmTheme

/** Small uppercase tracked label used above sections and inside cards. */
@Composable
fun WarmEyebrow(
    text: String,
    modifier: Modifier = Modifier,
    warmth: Warmth = Warmth.Subtle,
) {
    val palette = WarmTheme.palette(warmth)
    Text(
        text = text.uppercase(),
        modifier = modifier,
        style = TextStyle(
            fontSize = 11.sp,
            fontWeight = FontWeight.SemiBold,
            letterSpacing = (11 * 0.16).sp,
            color = palette.ink3,
        ),
    )
}
